run-dlp/README.md

```diff
@@ -0,0 +1,22 @@
+# run-dlp
```

Comment:
Since there's only one flow.png file, maybe we don't need "images" folder, just move it in the same folder as README?